### ISSUE TITLE
In-game key config setting "double tap space to toggle fly" requires restart to be applied

### ISSUE THREAD

**Issue by kaeza** [minetest/minetest#3043]
Toggling the "double tap space to toggle fly" checkbox in the in-game key config dialog is ignored.

Toggling the "use = climb down" checkbox works, and both checkboxes also work if toggled from the main menu key config dialog.

**Comment by est31** [comment]
I don't want to be bureaucratic, but the blocker label needs an explanation, why you consider the issue a blocker one. I just put the explanation @kwolekr gave on irc here:
```
<hmmmm> blatantly ignored settings like that is a blocker for sure
```

**Comment by est31** [comment]
fixed by a670ecfde4eccc0184089bac551b2c480ff35cda
